packages/react-shopify-app-bridge/src/Provider.tsx

```diff
@@ -1,10 +1,11 @@
 import type { AnyClient } from "@gadgetinc/api-client-core";
 import { Provider as GadgetUrqlProvider, useQuery } from "@gadgetinc/react";
 import type { History, LocationOrHref } from "@shopify/app-bridge-react";
-import AppBridgeReact from "@shopify/app-bridge-react";
+import AppBridgeProvider from "@shopify/app-bridge-react/components/Provider/Provider.js";
```

Comment:
Wow, that sucks. Offhand I don't know why, but this seems ok to me. It will be a bit more brittle in that if they change the layout inside the package this'll break but we can deal with that when it happens. 

You could also try `import * as AppBridgeReact from  "@shopify/app-bridge-react"`

Reply:
i'll give that a shot